<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HyunboStock</class>
 <widget class="QMainWindow" name="HyunboStock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1196</width>
    <height>677</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>1151</width>
      <height>651</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>주가정보</string>
     </attribute>
     <widget class="QTableWidget" name="tbl_stock_info">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>70</y>
        <width>1081</width>
        <height>481</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_stock_refresh">
      <property name="geometry">
       <rect>
        <x>1010</x>
        <y>560</y>
        <width>112</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>새로고침</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_stock_toExcel">
      <property name="geometry">
       <rect>
        <x>880</x>
        <y>560</y>
        <width>112</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>엑셀저장</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>221</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>맑은 고딕</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>최근 새로고침 시간</string>
      </property>
     </widget>
     <widget class="QLabel" name="txt_stock_refreshTime">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>30</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>맑은 고딕</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0000-00-00 00:00:00</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Investing.com</string>
     </attribute>
     <widget class="QTableWidget" name="tbl_investing_info">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>70</y>
        <width>1071</width>
        <height>531</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
       <property name="font">
        <font>
         <family>맑은 고딕</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Last</string>
       </property>
       <property name="font">
        <font>
         <family>맑은 고딕</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>High</string>
       </property>
       <property name="font">
        <font>
         <family>맑은 고딕</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Low</string>
       </property>
       <property name="font">
        <font>
         <family>맑은 고딕</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Chg.</string>
       </property>
       <property name="font">
        <font>
         <family>맑은 고딕</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Chg.%</string>
       </property>
       <property name="font">
        <font>
         <family>맑은 고딕</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Vol</string>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </column>
     </widget>
     <widget class="QComboBox" name="cmb_investing">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>20</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_investing_search">
      <property name="geometry">
       <rect>
        <x>990</x>
        <y>20</y>
        <width>112</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>SEARCH</string>
      </property>
     </widget>
     <widget class="QLabel" name="txt_time">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>10</y>
        <width>81</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>맑은 고딕</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Time</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="txt_investing_time">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>20</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>맑은 고딕</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0000-00-00</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>관심종목</string>
     </attribute>
     <widget class="QTableWidget" name="tbl_intrst_all">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>80</y>
        <width>430</width>
        <height>501</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="accessibleName">
       <string/>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="textElideMode">
       <enum>Qt::ElideMiddle</enum>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
      <column>
       <property name="text">
        <string>Ticker</string>
       </property>
       <property name="font">
        <font>
         <family>맑은 고딕</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>종목명</string>
       </property>
       <property name="font">
        <font>
         <family>맑은 고딕</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </column>
     </widget>
     <widget class="QTableWidget" name="tbl_intrst_intrst">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>80</y>
        <width>430</width>
        <height>501</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <column>
       <property name="text">
        <string>Ticker</string>
       </property>
       <property name="font">
        <font>
         <family>맑은 고딕</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>종목명</string>
       </property>
       <property name="font">
        <font>
         <family>맑은 고딕</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>40</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>모든 종목</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>40</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>관심 종목</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
